BUILD.bazel

```diff
@@ -12,26 +12,42 @@ cc_library(
     visibility = ["//visibility:public"],
 )
 
+cc_library(
```

Comment:
What is the logic for mapping a single file per library?  Seems these should all be part of a common "testing helpers" library but there may be something of the set-up Bazel I don't understand so wanted to clarify.

Reply:
There's no need for a library per file. Your suggestions works too.